src/gwc/src/main/java/org/geoserver/gwc/GWC.java

```diff
@@ -1248,18 +1254,6 @@ public GridSetBroker getGridSetBroker() {
         return gridSetBroker;
     }
 
-    // public GeoServerTileLayer getTileLayerById(final String id) {
-    // return embeddedConfig.getLayerById(id);
-    // }
-
-    public List<LayerInfo> getLayerInfos() {
-        return getCatalog().getLayers();
-    }
-
-    public List<LayerGroupInfo> getLayerGroups() {
-        return getCatalog().getLayerGroups();
-    }
-
```

Comment:
Yeah, I rather doubt anything is using them either, and I agree they are dangerous so removing them outright in master is fine by me. I would favour merely making them deprecated in a backport though, just in case someone somewhere has decided to use them in some custom extension, as unlikely as it may be,